src/main/java/com/qunite/api/web/dto/auth/AuthenticationResponse.java

```diff
@@ -2,9 +2,11 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Data
 @AllArgsConstructor
+@NoArgsConstructor
```

Comment:
For what reason?

Reply:
To readAccesToken in AccAuthenticationIntegrationTest.getAccessToken(). Besides that, I see no problem in dto having emptyConstructor.

Reply:
I do not think that adding code only and only for your tests is a good practise

Reply:
I do not think that adding noargaconstructor to dto is bad practice

Reply:
> I do not think that adding noargaconstructor to dto is bad practice

https://github.com/q-unite/qunite-api/pull/105/files#r1271322615